ical/event.py

```diff
@@ -282,11 +287,9 @@ def end_datetime(self) -> datetime.datetime:
     @property
     def computed_duration(self) -> datetime.timedelta:
         """Return the event duration."""
-        if self.end:
-            return self.end - self.start
-        if not self.duration:
-            raise ValueError("Invalid state, expected end or duration")
-        return self.duration
+        if self.duration != None:
```

Comment:
```suggestion
        if self.duration is not None:
```
prefering an explicit none check rather than comparison